api/serverless.yaml

```diff
@@ -21,7 +20,8 @@ provider:
   name: aws
   runtime: nodejs14.x
   region: us-east-1
-  stage: prod
+  stage: ${env:STACK_NAME}
+  stackName: peterportal-${env:STACK_NAME}
```

Comment:
Shouldn't this be `peterportal-core-${env:STACK_NAME}`? Not entirely sure how serverless works but this is based off the current stack name I see for your prod stack.

Reply:
Otherwise when you deploy to prod you'll make a new stack instead of updating the current one.

Reply:
Oh that’s right, I’ll go ahead and update that!

Reply:
Just fixed this so I’ll resolve this conversation